entrypoint.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
Does this entrypoint script need to be in the root of the repository and called entrypoint.sh? 

I can see that you are explicitly calling the file path in the workflow, I'd suggest to remove it from the root since it feels very invasive for a side, CI-related feature.